code/handlers/ldap.q

```diff
@@ -14,8 +12,8 @@ port:       @[value;`port;0i];                                     / port for ld
 blocktime:  @[value;`blocktime; 0D00:30:00];                       / time before blocked user can attempt authentication
 checklimit: @[value;`checklimit;3];                                / number of attempts before user is temporarily blocked
 checktime:  @[value;`checktime;0D00:05];                           / period for user to reauthenticate without rechecking LDAP server
-buildDNsuf: @[value;`buildDNsuf;""];                               / suffix used for building bind DN
-buildDN:    @[value;`buildDN;{{"uid=",string[x]}}]; / function to build bind DN
+buildDNsuf: @[value;`buildDNsuf;",ou=people,dc=planetexpress,dc=com"];                               / suffix used for building bind DN
```

Comment:
this is only the suffix for the test LDAP server we were using, default should be left empty